recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/pvr_debug_setup.sh

```diff
@@ -0,0 +1 @@
+pvrdebug -log trace -loggroups main,mts,cleanup,csw,bif,pm,rtd,spm,pow,hwr,hwp,rpm,dma,debug
```

Comment:
Why the file named *sh has not #!/bin/sh?

Reply:
default shell is used